### PR TITLE
Adds some more badges to the readme.

### PR DESCRIPTION
**Pull request by MrsTonedOne**
Found this, added it.

[Percentage of issues still open](http://isitmaintained.com/project/tgstation/tgstation "Percentage of issues still open") [Average time to resolve an issue](http://isitmaintained.com/project/tgstation/tgstation "Average time to resolve an issue")

### CODE REVIEW

**Comment by RemieRichards** [comment]
>4 days

fug.

**Comment by RemieRichards** [comment]
>projects recently analysed
>vg

was this you? :P

**Comment by MrStonedOne** [comment]
Yes, they had poor numbers.

**Comment by RemieRichards** [comment]
I saw lol.

**Comment by RemieRichards** [comment]
adding more badges makes me sad that codefactor only does real languages.

**Comment by Iamgoofball** [comment]
can we get a pornhub stamp of approval too?

**Comment by Cyberboss** [comment]
Need dis [forthebadge](http://forthebadge.com)

**Comment by RemieRichards** [comment]
[forthebadge](http://forthebadge.com)
[forthebadge](http://forthebadge.com)
[forthebadge](http://forthebadge.com)
[forthebadge](http://forthebadge.com)
[forthebadge](http://forthebadge.com)
[forthebadge](http://forthebadge.com)

needs these ones too

**Comment by Cyberboss** [comment]
I will actually put all my support for built with resentment

**Comment by Tacolizard** [comment]
Fuck it ship it and built with resentment embody this codebase so well it would be blasphemy to not include them

**Comment by RemieRichards** [comment]
Was going to PR the resentment badge to your branch but github won't:
A. allow me to PR to your batch branch (404's)
B. do it from this repo
C. my branch is ancient can't do it from there

so pls do it yourself @MrStonedOne

File: README.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 [![Build Status](https://travis-ci.org/tgstation/tgstation.png)](https://travis-ci.org/tgstation/tgstation) [![Krihelimeter](http://www.krihelinator.xyz/badge/tgstation/tgstation)](http://www.krihelinator.xyz)  
 [![Percentage of issues still open](http://isitmaintained.com/badge/open/tgstation/tgstation.svg)](http://isitmaintained.com/project/tgstation/tgstation "Percentage of issues still open") [![Average time to resolve an issue](http://isitmaintained.com/badge/resolution/tgstation/tgstation.svg)](http://isitmaintained.com/project/tgstation/tgstation "Average time to resolve an issue")
 
+[![forthebadge](http://forthebadge.com/images/badges/built-with-resentment.svg)](http://forthebadge.com)
```
Comment (RemieRichards) [inline README.md]: This looks bad, you were supposed to push it up one line (to line 5) and start it with a ```<BR>``` dumb smugfruitdog

Comment (optimumtact) [inline README.md]: 😢

**Pull request merged**